tests/test_parser.py

```diff
@@ -0,0 +1,278 @@
+from typing import Any
```

Comment:
The test coverage should be checked.

Reply:
The problem here is that the Python files under `language` are not used directly. Instead, `langkit` uses them as a DSL to generate a separate Python library. The coverage of this library is not great, as a lot of unused code is generated (including branches only relevant to Windows etc.).

I restored coverage analysis in bd1f32d and set the threshold to the currently achieved coverage (~71%) so we at least don't get worse.